src/controllers/trust.details.controller.ts

```diff
@@ -1,17 +1,58 @@
 import { NextFunction, Request, Response } from 'express';
 import * as config from '../config';
 import { logger } from '../utils/logger';
+import { getBoIndividualAssignableToTrust, getBoOtherAssignableToTrust } from '../utils/trusts';
 import { getApplicationData, setExtraData } from '../utils/application.data';
-import * as mapperToPageService from '../utils/trust/mapper.to.page';
-import * as mapperToSessionService from '../utils/trust/mapper.to.session';
+import * as mapperDetails from '../utils/trust/details.mapper';
 import { ApplicationData } from '../model/application.model';
 import * as PageModel from '../model/trust.page.model';
 import { Trust, TrustKey } from '../model/trust.model';
+import { BeneficialOwnerIndividualKey } from '../model/beneficial.owner.individual.model';
+import { BeneficialOwnerOtherKey } from '../model/beneficial.owner.other.model';
 
 const TRUST_DETAILS_TEXTS = {
   title: 'Tell us about the trust',
 };
 
+type TrustDetailPageProps = {
```

Comment:
Here and below, I'd have just gone for 'PageProperties' and not shortened it (we don't shorten 'Application' to 'App') but minor and up to you.

Reply:
fixed